py/circuitpy_mpconfig.mk

```diff
@@ -264,12 +269,12 @@ endif
 CFLAGS += -DCIRCUITPY_UHEAP=$(CIRCUITPY_UHEAP)
 
 ifndef CIRCUITPY_USB_HID
-CIRCUITPY_USB_HID = $(CIRCUITPY_DEFAULT_BUILD)
+CIRCUITPY_USB_HID = $(CIRCUITPY_ALWAYS_BUILD)
```

Comment:
How do these work if USB isn't available?

Reply:
I thought these didn't interrupt compilation, but on double checking it seems I messed up my macro test and they do halt things. I'll remove them.